apps/web-client/src/lib/db/cr-sqlite/db.ts

```
@@ -109,3 +109,67 @@ export const getPeerDBVersion = async (db: DB, siteId: Uint8Array): Promise<bigi
+ * NOTE: Currently it clears all data for the DB - this deletes all DBs created using the current cr-sqlite stack
+ * (e.g. we could have multiple DBs "dev", "foo", "bar-122", etc. created by cr-sqlite and all stored in "idb-batch-atomic" - this will delete them all)
+ * TODO: Implement a more fine-grained approach - delete only the chunks associated with a particular DB name, e.g. "dev"
+ */
+export async function clearDb() {
+	const dbName = "idb-batch-atomic";
+
+	const db = await openIndexedDB(dbName);
+	await clearIndexedDB(db);
+
+	// TODO: This is a bit inconsistent -- maybe clear only the "dev" db
+	delete dbCache["dev"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)